media/templates/embed.mustache

```diff
@@ -0,0 +1,281 @@
+{{#>>collection_table}}
```

Comment:
Because this is being rendered by `CollectionList`, which I've recently updated to use Mustache 2, this template will need some updates. In particular, using `{{#>>` to denote a partial won't work. I can help with this merge to the new code in the `fall.midterm.2015` branch if you like.